docs/UserGuide.md

```diff
@@ -279,6 +367,31 @@ The steps for this command are as follows:
 
 Hall-y's data is saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
 
+
+### 6.11 Adding a hall event : `add-event`
+
+You can add a hall event by using the `add-event` command.
+
+The steps for this command are as follows:
+
+1. Enter the add event command by typing `add-event n/NAME d/DESCRIPTION` into the input box
+2. Press enter
+3. The result box will show the new event created:
+![New event added](https://i.imgur.com/70Va3Dt.png)
+4. The events list will show the newly added event:
+<img src="https://i.imgur.com/mNjKNOT.png" width="250"/>
+
+
+### 6.12 Deleting a hall event : `delete-event`
+
+You can delete a hall event specified at an index by using the `delete-event` command followed by the index.
+
+The steps for this command are as follows:
+1. Enter the delete event command by typing `delete-event INDEX` into the input box. <br>with `INDEX` being the corresponding index of the specified event as seen on the events list
+2. Press enter
+3. The result box will show 'Deleted Event:' and the details of the deleted event: 
+![TODO: Placeholder](https://i.imgur.com/oCRZHbD.png)
```

Comment:
I think you can replace this with the actual image right?

Reply:
Alright, done!